### PR TITLE
Add proper rel to external links

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/gameserver/components/embed/applet/applet.vue [hunk 1]
```diff
@@ -18,13 +18,16 @@
 				<translate>You may use the button below to get it.</translate>
 			</p>
 
-			<a href="https://java.com/download" onclick="window.open( this.href ); return false">
+			<external-link
+				href="https://java.com/download"
+				onclick="window.open( this.href ); return false"
```
Comment (YLivay) [inline src/gameserver/components/embed/applet/applet.vue]: i know its unrelated, but shouldnt we just do target blank?

File: src/gameserver/components/embed/flash/flash.vue [hunk 2]
```diff
@@ -17,13 +17,16 @@
 				<translate>You may use the button below to get it.</translate>
 			</p>
 
-			<a href="https://www.adobe.com/go/getflash" onclick="window.open( this.href ); return false">
+			<external-link
+				href="https://www.adobe.com/go/getflash"
+				onclick="window.open( this.href ); return false"
```
Comment (YLivay) [inline src/gameserver/components/embed/flash/flash.vue]: same here, target blank?

File: src/gameserver/components/embed/silverlight/silverlight.vue [hunk 3]
```diff
@@ -23,7 +23,7 @@
 				<translate>You may use the button below to get it.</translate>
 			</p>
 
-			<a
+			<external-link
 				href="https://go.microsoft.com/fwlink/?LinkID=124807"
 				onclick="window.open( this.href ); return false"
```
Comment (YLivay) [inline src/gameserver/components/embed/silverlight/silverlight.vue]: and here

File: src/gameserver/components/embed/applet/applet.vue (hunk 1 above)
Comment (hworld) [reply src/gameserver/components/embed/applet/applet.vue]: This should be a prop on external-link, imo. <external-link popup> or something stupid

**Pull request merged**